src/orwell/Application.cpp

```diff
@@ -87,6 +89,11 @@ bool Application::ReadParameters(
 		oParam.m_commandLineParameters.m_agentPort = 9003;
 		ORWELL_LOG_DEBUG("by default, agent-port = " << oParam.m_commandLineParameters.m_agentPort);
 	}
+	if (not oParam.m_commandLineParameters.m_replierPort)
+	{
+		oParam.m_commandLineParameters.m_replierPort = 9004;
```

Comment:
What about putting these constants together at the same place? It would be good to easily find which ports are used by the server.